x-pack/legacy/plugins/ml/public/data_frame/components/job_details/job_details_form.tsx

```diff
@@ -180,6 +185,24 @@ export const JobDetailsForm: SFC<Props> = React.memo(({ overrides = {}, onChange
           isInvalid={jobIdExists}
         />
       </EuiFormRow>
+      <EuiFormRow
+        label={i18n.translate('xpack.ml.dataframe.jobDetailsForm.jobDescriptionLabel', {
+          defaultMessage: 'Job description',
+        })}
+      >
+        <EuiFieldText
+          placeholder="job description"
+          value={jobDescription}
+          onChange={e => setJobDescription(e.target.value)}
+          aria-label={i18n.translate(
+            'xpack.ml.dataframe.jobDetailsForm.jobDescriptionInputAriaLabel',
+            {
+              defaultMessage: 'Choose an optional job description.',
+            }
+          )}
+          isInvalid={jobIdExists}
```

Comment:
Is this line needed?

Reply:
Copy-paste error, thanks for catching it. Fixed in e73ba2d.